api/features/versioning/managers.py

```diff
@@ -20,13 +20,15 @@ def get_latest_versions_by_environment_id(self, environment_id: int) -> RawQuery
         """
         return self._get_latest_versions(environment_id=environment_id)
 
-    def get_latest_versions_by_environment_api_key(
-        self, environment_api_key: str
+    def get_latest_versions(
+        self, environment_api_key: str = None, environment_uuid: str = None
```

Comment:
This feels a bit odd / inconsistent to me, why not just add a new `get_latest_versions_by_uuid` method?